infrastructure/terraform/per_account/mgmt/main.tf

```diff
@@ -0,0 +1,33 @@
+resource "aws_resourcegroups_group" "resource_group" {
```

Comment:
Do we need account wide terraform for MGMT?

Reply:
We need to setup the root DNS.